### DISCUSSION TITLE
主页的“检测到您的页面展示可能受到浏览器插件影响，建议您将当前页面加入插件白名单 ”提示条

### DISCUSSION THREAD

**Discussion by SunsetMkt** [the1812/Bilibili-Evolved#3719] [综合讨论 / General]
`class`为`adblock-tips`，能否创建一个插件用于隐藏它？

**Comment by the1812** [comment]
这个提示条如其名, 开了 AdBlock 就会看见, 是不是应该反馈给 AdBlock (或者具体点是 EasyList China 规则)?

**Reply by lingxd** [reply]
你只要用AdBlock 把这个当作广告拦截了就好了。AdBlock 拦截提示

**Reply by CloudyEagle** [reply]
> 你只要用AdBlock 把这个当作广告拦截了就好了。AdBlock 拦截提示

我有加过，但也希望这边也当作广告来去除。
之前该脚本的去广告模块失效过，就在ADP中加过相关规则。
![Snip_2025-03-12_22-49-52](https://github.com/user-attachments/assets/c0a21ecf-f144-4857-ade3-9d4fe7a5c47c)

今天想去看下该脚本的去广告是否已恢复，看到之前写的去插件提示的规则，便验证下脚本是否有去除的能力，结果没有……
加上自己的搜索能力有限，没找到这个讨论，新开了一个 #5167

**Comment by lingxd** [comment]
你只要用AdBlock 把这个当作广告拦截了就好了。AdBlock 拦截提示

**Reply by ishikisiko** [reply]
甜菜！

**Reply by CloudyEagle** [reply]
我有加过，但也希望这边也当作广告来去除。
之前该脚本的去广告模块失效过，就在ADP中加过相关规则。
今天想去看下该脚本的去广告是否已恢复，看到之前写的去插件提示的规则，便验证下脚本是否有去除的能力，结果没有……
加上自己的搜索能力有限，没找到这个讨论，新开了一个 #5167

**Reply by pencilqaq** [reply]
> 我有加过，但也希望这边也当作广告来去除。 之前该脚本的去广告模块失效过，就在ADP中加过相关规则。 今天想去看下该脚本的去广告是否已恢复，看到之前写的去插件提示的规则，便验证下脚本是否有去除的能力，结果没有…… 加上自己的搜索能力有限，没找到这个讨论，新开了一个 #5167

既然有解决方式就不要做更多的工作，用广告屏蔽插件才会有这个提示且广告屏蔽插件能屏蔽这个提示 那为什么还要再做一遍？
